zellij-server/src/wasm_vm.rs

```diff
@@ -248,6 +256,20 @@ fn host_set_timeout(plugin_env: &PluginEnv, secs: f64) {
     });
 }
 
+fn host_exec_cmd(plugin_env: &PluginEnv) {
+    let mut cmdline: Vec<String> = wasi_read_object(&plugin_env.wasi_env);
+    let command = cmdline.remove(0);
+
+    process::Command::new(command)
+        .args(cmdline)
+        .spawn()
```

Comment:
Just leave a note here, to avoid block plugin thread, we don't wait for the command to exit.